Openshift-EE/pipelines/Prometheus/stages/3-functional/ELHN-deploy-prometheus/prometheus-provision

```diff
@@ -65,18 +83,59 @@ run_test=apps/prometheus/loadgen/run_litmus_test.yml
 
 cat $run_test
 
+# Run the Litmus job and get the details of the litmus job from litmus_job_runner utils.
 bash ../Openshift-EE/utils/litmus_job_runner label='app:prometheus-loadgen-litmus' job=$run_test 
 
+
 echo "Dumping state of cluster post job run"; echo ""
 cd ..
+#Get the cluster state Once the litmus jobs completed.
 bash Openshift-EE/utils/dump_cluster_state;
+#Update the e2e event for the job
 bash Openshift-EE/utils/event_updater jobname:prometheus-provision $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
 
+# Update the result of the test case in github mayadata-io/e2e-openshift repository.
 if [ "$?" != "0" ]; then
 #python3 Openshift-EE/utils/result/result_update.py $job_id ELHN 3-functional "Deploying prometheus loadgen application" Fail $pipeline_id "$current_time" $commit_id $gittoken
 exit 1;
 fi
 
+################
+# LitmusBook 3 #
+################
+
+echo "*******Deploying Prometheus Liveness Job****"
+#kubectl apply -f Openshift-EE/utils/result-prometheus-liveness.yml
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=prometheus-liveness metadata="")
+echo $test_name
+
+cd litmus
+
+echo "Running the litmus test.."
+run_test=apps/prometheus/liveness/run_litmus_test.yml
+
+cat $run_test
+
+# Run the Litmus job and get the details of the litmus job from litmus_job_runner utils.
+bash ../Openshift-EE/utils/litmus_job_runner label='app:litmus-prometheus-liveness' job=$run_test 
+
+
+echo "Dumping state of cluster post job run"; echo ""
+cd ..
+#Get the cluster state Once the litmus jobs completed.
+bash Openshift-EE/utils/dump_cluster_state;
+#Update the e2e event for the job
+bash Openshift-EE/utils/event_updater jobname:prometheus-provision $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+# Update the result of the test case in github mayadata-io/e2e-openshift repository.
+if [ "$?" != "0" ]; then
+#python3 Openshift-EE/utils/result/result_update.py $job_id ELHN 3-functional "Deploying prometheus liveness job" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+# Update the e2e cr once the job is completed
+
 current_time=$(eval $time)
 bash Openshift-EE/utils/e2e-cr jobname:prometheus-provision jobphase:Completed end_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" test_result:Pass
```

Comment:
@harshita-sharma011  Instead of passing the test result status as `Pass` can you get the testresult status from litmusresult
for ex :
testResult=$(kubectl get litmusresult ${test_name} --no-headers -o custom-columns=:spec.testStatus.result)